app/build.gradle

```diff
@@ -101,7 +101,7 @@ ext {
     checkstyleVersion = '9.2.1'
 
     androidxLifecycleVersion = '2.3.1'
-    androidxRoomVersion = '2.3.0'
+    androidxRoomVersion = '2.4.1'
```

Comment:
Changelog: https://developer.android.com/jetpack/androidx/releases/room

Reply:
I'll be running this patch backported to master in the coming days.